app/helpers/routes/sessionHelper.js

```diff
@@ -0,0 +1,15 @@
+export const getFromSessionOrApi = async ({
```

Comment:
oooh I like this. Something we'll be able to start using for calls that follow this pattern 👍